public/js/components/SplitBox.css

```diff
@@ -14,7 +14,7 @@
 .split-box .controlled {
   display: flex;
   overflow: auto;
-  transition: width 0.5s ease;
+  /*transition: width 0.5s ease;*/
```

Comment:
It was very smooth for me before, so I wonder what's different. There shouldn't be any easing here though, right?